extensions/amp-auto-ads/0.1/ad-network-config.js

```diff
@@ -65,16 +67,16 @@ class AdSenseNetworkConfig {
 
   /** @override */
   getConfigUrl() {
+    const docInfo = documentInfoForDoc(this.autoAmpAdsElement_);
+    const canonicalHostname = parseUrl(docInfo.canonicalUrl).hostname;
```

Comment:
Why not use the `source` (the amp-doc's location) vs the canonical (the main document's location)?